docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -584,17 +586,19 @@ relevant schema and table names supplied with the required parameters
 `schema_name` and `table_name`:
 
 ```sql
-ALTER TABLE testdb.iceberg_customer_orders EXECUTE add_files_from_table(
+ALTER TABLE testdb.iceberg_customer_orders 
+EXECUTE example.system.add_files_from_table(
```

Comment:
The intro sentence talks about the the requirement for the example catalog .. but it is never actually fulfilled.. question would be .. does a use statement before this query work and then it works without example.system ?

Reply:
It works without `example.system` when the default session catalog is `example`.

Reply:
Added that info.